### ISSUE TITLE
HtmlField and HtmlBodyPart, UI localization 

### ISSUE THREAD

**Issue by PiemP** [OrchardCMS/OrchardCore#7295]
Hi all, 

I have see in the `HtmlField-Trumbowyg.Edit.cshtml` that exist the configuration that tell to trumbowyg which kind of language have to display. But I see that missings trumbowyg's localization files in the `OrchardCore.Resources` module. Is there a plan to introduce this files in `OrchardCore.Resources`?
Thank you.

**Comment by hishamco** [comment]
I don't think it's a good idea to include all the Trumbowyg translation, but mention this in docs at least it will add a value, so the user can know how to localize the Trumbowyg

@agriffard your though

**Comment by sebastienros** [comment]
I agree that a guide to show how to download the required files and also to do any necessary changes that would be required in our templates.

**Comment by PiemP** [comment]
What is the best approach to do it? Create an admin theme and add somewhere the trumbowyg translation file?

Thank you

**Comment by hishamco** [comment]
@PiemP if you want to add the docs for this will be better, otherwise I will try to add it if I have a time, coz I need to update some other PRs that have higher priority than this
